iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/unique_ptr.inl

```diff
@@ -48,7 +52,7 @@ unique_ptr<T>& unique_ptr<T>::operator=(unique_ptr&& rhs) noexcept
     }
     return *this;
 }
-
+// AXIVION Next Construct AutosarC++19_03-A12.6.1 : m_deleter is initialized in constructor
```

Comment:
This rational is not complete in my opinion.
* `m_deleter` was initialized in the constructor in line 32
* every move operation is using this value to initialize their members in line 51

Reply:
@elfenpiff Agree, it is a False Positive.

Reply:
@elfenpiff @saif-at-github Is there another way to call a ctor or copy ctor without initializing this? It can be initialized as the null-function by the ctor (problematic), but this will not trigger the rule as the object would be still initialized.

As long as it is guaranteed it is initialized in all cases of construction, we can state this as justification (did not check whether this is the case but found no simple counterexample either).